molecule/default/files/etc-nginx/conf.d/proxy-other.conf

```diff
@@ -36,14 +36,9 @@ server {
         return 302 $redirect_uri;
     }
 
-    location /direct {
```

Comment:
Can you explain why these get dropped?

Reply:
I had the same question initially and I think this is because the `molecule` test overrides the `default_nginx_proxy_direct_locations` to test the functionality in this PR. I am not sure testing the default configuration is worth it ?

Reply:
As @sbesson says, it's because this virtualhost overrides the default `default_nginx_proxy_direct_locations`:
- https://github.com/ome/ansible-role-nginx-proxy/tree/1.11.0#role-variables-multiple-sites
- https://github.com/ome/ansible-role-nginx-proxy/blob/1.11.0/templates/nginx-confd-proxy.j2#L87

Reply:
:+1: Thanks. As long as testing that case is no longer important, gfi.